README.md

```diff
@@ -85,50 +103,45 @@ Replace the variables with the one given in the UI:
 </script>
 ```
 
-This will automatically add an instant search autocomplete to your website using your newly created Algolia index.
-Please refer to the [full documentation](https://github.com/algolia/algoliasearch-netlify/tree/master/frontend) to configure this frontend plugin.
-
-> Screenshot
```

Comment:
I don't know if it was forgotten or not but in the meantime I removed it.

Reply:
It's been added since you've started your review.

Reply:
Ouch! Fixed it in rebase f01f8d5.